README.md

```diff
@@ -442,12 +442,16 @@ A curated list of awesome Java frameworks, libraries and software.
 
 *Everything else.*
 
+* [Avian](https://github.com/ReadyTalk/avian) - Open source JVM with both a JIT & AOT modes. Includes an iOS port.
```

Comment:
Remove the "Open source", see comment in Conversations. Also we have a JVM category, move the entry.